.travis.yml

```diff
@@ -67,23 +69,23 @@ install:
   - travis_retry ./mvnw -v
   - |
     if [[ -v TEST_SPECIFIC_MODULES ]]; then
-      ./mvnw install $MAVEN_FAST_INSTALL -pl "${TEST_SPECIFIC_MODULES}" -am
+      env MAVEN_OPTS="${MAVEN_INSTALL_OPTS}" ./mvnw install $MAVEN_FAST_INSTALL -pl "${TEST_SPECIFIC_MODULES}" -am
```

Comment:
I've always done `FOO=xxx <command>` to set environment variables before calling a command. Is `env FOO=xxx <command>` different from that? Maybe a more portable way?

Reply:
`FOO=xxx` is interpreted by shell (`sh` here), `env` is a program.
both behave effectively the same.

we use `env` already and IMO it increases readability, highlighting the fact the env var is set for one command only

https://github.com/prestosql/presto/blob/d2630891726cbea623abb9aea3935e5dfceb66d6/.travis.yml#L138-L151